### PR TITLE
[WM-2013] Environment setup updates

### PR DESCRIPTION
**Pull request by dillydally414**

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -44,11 +47,16 @@ The Batch Analysis service relies on a Postgresql database server. There are two
...
+Make sure to use the correct version, as noted above.
+
 #### Initialize your database:
 ```sh
 psql -h 127.0.0.1 -U postgres -f ./common/postgres-init.sql
 ```
-***N.B.*** If you used **the convenient app**, you should run `psql` as `/Applications/Postgres.app/Contents/Versions/latest/bin/psql`
+***N.B.*** If you used **the convenient app**, you should run `psql` as `/Applications/Postgres.app/Contents/Versions/13/bin/psql`. To add Postgres commands to your path, run the following command:
+```sh
+sudo mkdir -p /etc/paths.d && echo /Applications/Postgres.app/Contents/Versions/13/bin | sudo tee /etc/paths.d/postgresapp
```
Comment (cjllanwarne) [inline DEVELOPMENT.md:58]: Huh, TIL about this. I've always just added `PATH=$PATH:.....` into bashrc files before

File: DEVELOPMENT.md [hunk 2]
```diff
@@ -65,29 +73,30 @@ CREATE DATABASE wds;
 Start a WDS container with the following command:
 Replace [YOUR_USERNAME] with something like mspector
 
-```
+```sh
```
Comment (cjllanwarne) [inline DEVELOPMENT.md:76]: Nice.

File: DEVELOPMENT.md [hunk 3]
```diff
@@ -141,7 +166,8 @@ Then navigate to the Swagger: `http://localhost:8080/swagger-ui.html`
 - Check out [gng](https://github.com/gdubw/gng), it'll save you typing `./gradlew` over
   and over, and also takes care of knowing when you're not in the root directory so you
   don't have to figure out the appropriate number of `../`s.
-- In IntelliJ, instead of running the local server with `bootRun`, use the `TerraBatchAnalysisApplication` Spring
-  Boot configuration that IntelliJ auto-generates. To edit it, click on it (in the upper
+- In IntelliJ, instead of running the local server with `bootRun`, use the Spring
+  Boot configuration that IntelliJ auto-generates (`-cp cbas.service.main`, main class
+  `bio.terra.cbas.App`). To edit it, click on it (in the upper
```
Comment (cjllanwarne) [inline DEVELOPMENT.md:172]: My intellij run mode has been broken for a little while and I'm not 100% sure why. I've just been using `../gradlew bootRun` but this encouraged me to try again. And... recreating the run mode with this configure just worked. Woohoo!

**Review by cjllanwarne — APPROVED** [review]
💯

**Review by kpierre13 — APPROVED** [review]

File: DEVELOPMENT.md (hunk 1 above)
Comment (dillydally414) [reply DEVELOPMENT.md:58]: Copied straight from [Postgres setup](https://postgresapp.com/). I usually do the same thing 🤷‍♂️

**Pull request merged**